README.md

```diff
@@ -434,7 +432,4 @@ First run `$ bundle` (requires the bundler gem), then...
 Note: run `$ rake -T` to get a list of all the available rake tasks.
 
 ## Authors
- - Jared Smartt - [@jsmartt](https://github.com/jsmartt)
- - Henrique Diomede - [@hdiomede](https://github.com/hdiomede)
- - Thiago Miotto - [@tmiotto](https://github.com/tmiotto)
- - Ricardo Piantola - [@piantola](https://github.com/piantola)
+See the [contributors graph](graphs/contributors)
```

Comment:
It's working well, @aalexmonteiro .
You can test it selecting the release4.1.0 branch, in home url of this repo. See [here](https://github.com/HewlettPackard/oneview-sdk-ruby/tree/release4.1.0)